coffee_cmd/src/cmd.rs

```diff
@@ -73,6 +73,12 @@ pub enum CoffeeCommand {
     /// tipping a plugins developer.
     #[clap(arg_required_else_help = false)]
     Tip { plugin: String, amount_msat: u64 },
+    /// Disable a plugin
+    #[clap(arg_required_else_help = true)]
+    Disable { plugin: String },
+    /// Enable a plugin
+    #[clap(arg_required_else_help = true)]
+    Enable { plugin: String },
```

Comment:
I was more happy if the enable and disable command was on two different commit

Reply:
That would have been better, yes.